packages/theme/composables/useAddresses/index.ts

```diff
@@ -115,15 +44,15 @@ export function useAddresses(): UseAddressesInterface {
     return results;
   };
 
-  const save = async (saveParams: ComposableFunctionArgs<UseAddressesParamsInput>) => {
-    Logger.debug('[Magento] save user address:', saveParams.address);
+  const save = async (params: ComposableFunctionArgs<UseAddressesParamsInput>) => {
```

Comment:
```suggestion
  const save = async (params?: ComposableFunctionArgs<UseAddressesParamsInput>) => {
```
Otherwise I will be forced to pass something like a `{}`

Reply:
Places like `params.prop` should be also updated